### PR TITLE
docs - wikitext-macros new v5.3.x syntax

### PR DESCRIPTION
**Pull request by pmario**
@Jermolene This is a docs only PR:

- wikitext-macros new v5.3.x syntax

### CODE REVIEW

**Comment by Jermolene** [comment]
Thanks @pmario

**Comment by Jermolene** [comment]
Hi @pmario this seems to be the cause of the bug in #8058 so I have reverted it for now

File: editions/tw5.com/tiddlers/system/wikitext-macros.tid [hunk 1]
```diff
@@ -1,104 +1,96 @@
...
 [[$tiddler$]]:
-
 <<tw-code $tiddler$>>
 \end
 
-\define flex-card(class,bordercolor:"",backgroundcolor:"",textcolor:"",imageField:"image",captionField:"caption",subtitle:"",descriptionField:"description",linkField:"link")
-\whitespace trim
-<$link class={{{ [<__class__>addprefix[tc-card ]] }}} to={{{ [<currentTiddler>get<__linkField__>else<currentTiddler>] }}}>
-	<div class="tc-card-accent" style.borderTop={{{ [<__bordercolor__>!is[blank]addprefix[5px solid ]] }}} style.background={{!!background}} style.backgroundColor=<<__backgroundcolor__>> style.color=<<__textcolor__>> style.fill=<<__textcolor__>>>
+\procedure flex-card(class,bordercolor:"",backgroundcolor:"",textcolor:"",imageField:"image",captionField:"caption",subtitle:"",descriptionField:"description",  inkField:"link")
```
Comment (BramChen) [inline editions/tw5.com/tiddlers/system/wikitext-macros.tid:91]: @Jermolene and @pmario
there is a wrong parameter name `inkField`, it should be `linkField`

Comment (pmario) [reply editions/tw5.com/tiddlers/system/wikitext-macros.tid:91]: @BramChen -- thanks for the info. I did fix it with pr https://github.com/Jermolene/TiddlyWiki5/pull/8059. I did probably hit the delete button too often ;)

**Pull request merged**